src/components/utils/OSMImporter.vue

```diff
@@ -150,6 +152,7 @@ export default {
     >
       <template v-slot:activator="{ on, attrs }">
         <v-btn
+          :disabled="!isUserSignedIn"
```

Comment:
changer le popup       <span>{{ $gettext("Download OSM network") }}</span> aussi.
login to download OSM network